Tests/ToolBox/ForexVolume/FxcmVolumeDownloaderTest.cs

```diff
@@ -137,7 +136,7 @@ public void RequestWithMoreThan10KMinuteObservationIsCorrectlySaved()
             Assert.AreEqual(expected: 27, actual: files.Length);
         }
 
-        [Ignore("Long test")]
+//        [Ignore("Long test")]
```

Comment:
are these intended to remain commented out? I suspect we want these to **not** run on the ci server

Reply:
I forgot to uncomment it.

I forgot to uncomment it, the good news are they passed 👍